Revert "build(deps-dev): bump autoprefixer from 9.8.6 to 10.3.1"

premanshup opened the pull request:
Reverts brainstormforce/astra#2987

Fix: Grunt minify not working due to incompatibility with autoprefixer latest version and post-css current version.

![MINGW64__c_xampp_htdocs_css-revamp_wp-content_themes_astra 2021-07-19 at 12 27 48 PM](https://user-images.githubusercontent.com/36841355/126116438-344b4df6-532d-406f-8b39-db6156805fb9.jpeg)

Nikschavan commented:
@premanshup Let's try to update postcss as well then?

btw - This looks like an error from postcss-flexibility - https://github.com/7rulnik/postcss-flexibility/issues/145#issuecomment-880573846

I have this fixed in a test branch at (https://github.com/Nikschavan/postcss-flexibility/tree/fix-145-build), you can run - `npm install git://github.com/Nikschavan/postcss-flexibility#fix-145-build --save-dev` which will install this updated dependency

premanshup commented:
> @premanshup Let's try to update postcss as well then?
> 
> btw - This looks like an error from postcss-flexibility - [7rulnik/postcss-flexibility#145 (comment)](https://github.com/7rulnik/postcss-flexibility/issues/145#issuecomment-880573846)
> 
> I have this fixed in a test branch at (https://github.com/Nikschavan/postcss-flexibility/tree/fix-145-build), you can run - `npm install git://github.com/Nikschavan/postcss-flexibility#fix-145-build --save-dev` which will install this updated dependency

@Nikschavan there is no update on https://www.npmjs.com/package/grunt-postcss. Thank you for the information, I'm checking this with the fix you have added for `postcss-flexibility`.

Nikschavan commented:
@premanshup you can switch grunt-postcss to [@lodder/grunt-postcss](https://www.npmjs.com/package/@lodder/grunt-postcss) as mentioned in the https://github.com/7rulnik/postcss-flexibility/issues/145#issuecomment-880573846, that should work as well, this dependency looks more maintained.

premanshup commented:
> @premanshup you can switch grunt-postcss to [@lodder/grunt-postcss](https://www.npmjs.com/package/@lodder/grunt-postcss) as mentioned in the [7rulnik/postcss-flexibility#145 (comment)](https://github.com/7rulnik/postcss-flexibility/issues/145#issuecomment-880573846), that should work as well, this dependency looks more maintained.

@Nikschavan It looks like the error is coming with `grunt-postcss` as well. I have added 
`"postcss-flexibility": "git://github.com/Nikschavan/postcss-flexibility.git#fix-145-build"`
but still getting the same error.
Need to check if there will be any changes in the
`wp-content\themes\astra\Gruntfile.js` if we switch `grunt-postcss` to `@lodder/grunt-postcss`

I have added a task for this update - https://wp-astra.atlassian.net/browse/AST-705.

I think for now, we can merge this so that we can start working with the existing PRs, I will check this once PRs are merged.
